src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/dialogs/project/pages/MavenGradleNewProjectFinalPage.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.project.pages;
+
+import com.intellij.remoterobot.RemoteRobot;
+import com.intellij.remoterobot.data.RemoteComponent;
+import com.intellij.remoterobot.fixtures.ContainerFixture;
+import com.intellij.remoterobot.fixtures.DefaultXpath;
+import com.intellij.remoterobot.fixtures.FixtureName;
+import com.redhat.devtools.intellij.commonUiTestLibrary.utils.labels.ButtonLabels;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+import static com.intellij.remoterobot.search.locators.Locators.byXpath;
+
+/**
+ * New Project dialog maven project second page fixture
+ *
+ * @author zcervink@redhat.com
+ */
+@DefaultXpath(by = "MyDialog type", xpath = "//div[@class='DialogRootPane']")
+@FixtureName(name = "New Project Dialog")
+public class MavenGradleNewProjectFinalPage extends AbstractNewProjectFinalPage {
+    public MavenGradleNewProjectFinalPage(@NotNull RemoteRobot remoteRobot, @NotNull RemoteComponent remoteComponent) {
+        super(remoteRobot, remoteComponent);
+    }
+
+
+    /**
+     * Open the 'Artifact Coordinates' options
+     */
+    public void openArtifactCoordinates() {
+        List<ContainerFixture> cf = findAll(ContainerFixture.class, byXpath("//div[@class='HideableTitledSeparator']/../*"));
+        boolean isAlreadyOpened = cf.size() > 5;
```

Comment:
How exactly is this ever going to evaluate to true? There is only a single `HideableTitledSeparator` at any time.

Reply:
Please look carefully on the xPath - it is not only the **HideableTitledSeparator**, but there is **/../*** at the end. 2 dots mean one level up, slash asterisk means all children. This tests whether the settings is opened. The metod opens the settings if it is closed by clicking on on label. When the settings is opened it will not click again because that means that the settings will be closed.

Reply:
The setting is always closed at the beginning of all tests. But when you open the settings -> close New Project dialog -> open new Project dialog -> it remains opened. This is a robust way to call method for opening the setting before editing the settings and if happens that it is already opened it will not click again on the settings label to prevent closing the settings.

Reply:
Are you telling me there is no way to retrieve attributes of an element? `extractData` method maybe?

Reply:
@jrichter1  this is probably the last change request from you - what do you think? In my opinion we have no other simple way to do that, so do we leave it as it is.

Reply:
xPath updated. Fixed.